sdk/flow/request.go

```diff
@@ -85,6 +83,8 @@ func MarshalChan(replyc chan *grpcflow.Reply, newItem func() proto.Message) (cha
 				continue
 			}
 
+			item = setDefaults(item)
```

Comment:
why not `itemc <- setDefaults(item)` ?

Reply:
Are you suggesting another method like `MarshalChan` that sits downstream? Like `completeItemC<- setDefaults(itemc)`?

Reply:
Oh, yeah, gotcha, I can just do it in one line haha